lib/mayaUsd/ufe/Utils.h

```diff
@@ -116,11 +116,6 @@ PXR_NS::UsdTimeCode getTime(const Ufe::Path& path);
 MAYAUSD_CORE_PUBLIC
 PXR_NS::TfTokenVector getProxyShapePurposes(const Ufe::Path& path);
 
-//! Check if the src and dst attributes are connected.
-//! \return True, if they are connected.
-MAYAUSD_CORE_PUBLIC
-bool isConnected(const PXR_NS::UsdAttribute& srcUsdAttr, const PXR_NS::UsdAttribute& dstUsdAttr);
```

Comment:
Moving it to `usdUfe/ufe` as I see this function doesn't need the maya and mayaUsd .h included in `lib/mayaUsd/ufe/Utils.h`